x-pack/plugins/enterprise_search/public/applications/workplace_search/components/layout/personal_dashboard_layout/personal_dashboard_layout.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiPage, EuiPageSideBar, EuiPageBody, EuiCallOut } from '@elastic/eui';
+
+import { AccountHeader } from '..';
+
+import { PRIVATE_DASHBOARD_READ_ONLY_MODE_WARNING } from '../../../views/content_sources/constants';
+
+import './personal_dashboard_layout.scss';
+
+interface LayoutProps {
+  restrictWidth?: boolean;
+  readOnlyMode?: boolean;
+  sidebar: React.ReactNode;
+}
+
+export const PersonalDashboardLayout: React.FC<LayoutProps> = ({
+  children,
+  restrictWidth,
+  readOnlyMode,
+  sidebar,
+}) => {
+  return (
+    <>
+      <AccountHeader />
+      <EuiPage className="enterpriseSearchLayout personalDashboardLayout">
+        <EuiPageSideBar className="enterpriseSearchLayout__sideBar personalDashboardLayout__sideBar">
+          {sidebar}
+        </EuiPageSideBar>
+        <EuiPageBody className="enterpriseSearchLayout__body" restrictWidth={restrictWidth}>
```

Comment:
[Optional] I found that WS universally sets `restrictWidth` on its layouts, and since this is WS-only layout, you might as well just always have it be true

```suggestion
        <EuiPageBody className="enterpriseSearchLayout__body" restrictWidth>
```

(+ removing the prop + typing above - no worries if you'd rather punt on this, I can grab it in my KibanaPageTemplate work)

Reply:
I like the suggestion! But I would rather punt on this. Unless you do it in your PR, I will add it to the polish list. Let me know what you decide.

Reply:
I'll do it in my PR, no worries!! 🎉